lib/QuipProcessor.js

```diff
@@ -295,8 +295,26 @@ class QuipProcessor {
                 dateStr = updatedDate === createdDate? createdDate : `${createdDate} (Updated: ${updatedDate})`;
                 dateStr = `, ${dateStr}`;
             }
+
+            let highlightedSections = "";
+            if (
+                message.annotation &&
+                message.annotation.id &&
+                message.annotation.highlight_section_ids
```

Comment:
I noticed there are some annotations without this field, I don't yet know what they represent though. From reading the [API docs](https://quip.com/dev/automation/documentation/current#tag/Messages) I thought that `annotation.id` should return the section anchor, but for whatever reason it seems to return something slightly different that I could not find in the documents referenced, so I assumed that it was not relevant. Maybe it happens for resolved comment threads? Not sure yet...